com.unity.netcode.gameobjects/Tests/Editor/NetworkObjectTests.cs

```diff
@@ -23,8 +23,9 @@ public void NetworkManagerOverrideTest()
             // Set override
             networkObject.NetworkManagerOwner = networkManager;
 
-            Debug.Assert(networkObject.NetworkManager == networkManager);
-
+            // Debug.Assert(networkObject.NetworkManager == networkManager);
```

Comment:
not a fan of commenting out tests. They used to work, why don't they anymore? Is there a fix needed?
As a last resort, we should [ignore] tag these tests, but commenting them just makes them disappear with no way of tracking them later

Reply:
Not sure how this slipped in, it was only supposed to have been there as part of debugging, my error, I'll fix it shortly.